Sources/PointFree/Auth.swift

```diff
@@ -0,0 +1,123 @@
+import Either
+import Foundation
+import Html
+import HttpPipeline
+import HttpPipelineHtmlSupport
+import Optics
+import Prelude
+
+let secretHomeResponse: (Conn<StatusLineOpen, Prelude.Unit>) -> IO<Conn<ResponseEnded, Data?>> =
+  writeStatus(.ok)
+    >>> readGitHubSessionCookieMiddleware
+    >>> respond(secretHomeView)
+    >>> pure
```

Comment:
will be able to get rid of `pure` and use `>->` if we merge https://github.com/pointfreeco/swift-web/pull/58